### PR TITLE
Add ROCm blurbs to linux installation instructions

### PR DESCRIPTION
**Pull request by jithunnair-amd**
@jeffdaily @sunway513 please review

### CODE REVIEW

File: _get_started/installation/linux.md [hunk 1]
```diff
@@ -80,28 +80,37 @@ sudo apt install python3-pip
 ### Anaconda
 {: #linux-anaconda}
 
-#### No CUDA
+#### No CUDA/ROCm
 
-To install PyTorch via Anaconda, and do not have a [CUDA-capable](https://developer.nvidia.com/cuda-zone) system or do not require CUDA, in the above selector, choose OS: Linux, Package: Conda and CUDA: None.
+To install PyTorch via Anaconda, and do not have a [CUDA-capable](https://developer.nvidia.com/cuda-zone) or [ROCm-capable](https://docs.amd.com) system or do not require CUDA/ROCm (ie. GPU support), in the above selector, choose OS: Linux, Package: Conda, Language: Python and Compute Platform: CPU.
```
Comment (jeffdaily) [inline _get_started/installation/linux.md:85]:
```suggestion
To install PyTorch via Anaconda, and do not have a [CUDA-capable](https://developer.nvidia.com/cuda-zone) or [ROCm-capable](https://docs.amd.com) system or do not require CUDA/ROCm (i.e. GPU support), in the above selector, choose OS: Linux, Package: Conda, Language: Python and Compute Platform: CPU.
```

File: _get_started/installation/linux.md [hunk 2]
```diff
@@ -80,28 +80,37 @@ sudo apt install python3-pip
...
+PyTorch via Anaconda is not supported on ROCm currently. Please use pip instead.
+
 
 ### pip
 {: #linux-pip}
 
 #### No CUDA
 
-To install PyTorch via pip, and do not have a [CUDA-capable](https://developer.nvidia.com/cuda-zone) system or do not require CUDA, in the above selector, choose OS: Linux, Package: Pip and CUDA: None.
+To install PyTorch via pip, and do not have a [CUDA-capable](https://developer.nvidia.com/cuda-zone) or [ROCm-capable](https://docs.amd.com) system or do not require CUDA/ROCm (ie. GPU support), in the above selector, choose OS: Linux, Package: Pip, Language: Python and Compute Platform: CPU.
```
Comment (jeffdaily) [inline _get_started/installation/linux.md:103]:
```suggestion
To install PyTorch via pip, and do not have a [CUDA-capable](https://developer.nvidia.com/cuda-zone) or [ROCm-capable](https://docs.amd.com) system or do not require CUDA/ROCm (i.e. GPU support), in the above selector, choose OS: Linux, Package: Pip, Language: Python and Compute Platform: CPU.
```

**Review by jeffdaily — CHANGES_REQUESTED** [review]
Two nit changes requested.  Otherwise LGTM.

**Comment by jithunnair-amd** [comment]
cc @atalman @malfet Who should be added as reviewer for this PR?

**Pull request merged**